src/overlays/actors/ovl_En_And/z_en_and.h

```diff
@@ -2,16 +2,37 @@
 #define Z_EN_AND_H
 
 #include "global.h"
+#include "objects/object_and/object_and.h"
 
 struct EnAnd;
 
 typedef void (*EnAndActionFunc)(struct EnAnd*, PlayState*);
 
 typedef struct EnAnd {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x44];
+    /* 0x144 */ SkelAnime skelAnime;
     /* 0x188 */ EnAndActionFunc actionFunc;
-    /* 0x18C */ char unk_18C[0x188];
+    /* 0x18C */ u8 action;
+    /* 0x18D */ char pad18D[0x7];
```

Comment:
```suggestion
    /* 0x18D */ UNK_TYPE1 pad18D[0x7];
```
With leftover unknown struct members, we like to change the type to `UNK_TYPE1` instead of leaving it `char`.